MyKSuite/src/main/java/com/infomaniak/core/myksuite/ui/views/BaseMyKSuiteChipView.kt

```diff
@@ -46,6 +46,6 @@ abstract class BaseMyKSuiteChipView @JvmOverloads constructor(
     @Composable
     override fun Content() {
         val colorRes = if (backgroundColor == ResourcesCompat.ID_NULL) null else backgroundColor
-        MyKSuiteTheme { MyKSuiteChip(tier = tier, backgroundColor = colorRes) }
+        MyKSuiteXMLTheme { MyKSuiteChip(tier = tier, backgroundColor = colorRes) }
```

Comment:
This is more on the anecdotic side of things but `backgroundColor` could be called `backgroundColorRes` and have a `@ColorRes` anontation. Also you could've used `takeIf` as in `val colorRes = backgroundColor.takeIif { it != ResourcesCompat.ID_NULL}`